deploy/deployment_dev.yaml

```
@@ -56,4 +56,6 @@ spec:
     kind: Service
     name: registration-service
     weight: 100
+  tls:
+    termination: edge 
```

It didn't pass the linter check:
```
yamllint -c .yamllint ./deploy/deployment_dev.yaml ./deploy/role.yaml ./deploy/role_binding.yaml ./deploy/service_account.yaml
./deploy/deployment_dev.yaml
  60:22     error    trailing spaces  (trailing-spaces)

make: *** [lint-yaml] Error 1
```
```suggestion
    termination: edge
```

---

Fixed in https://github.com/codeready-toolchain/registration-service/pull/57/commits/f99aab9fcb9d08120e92a5b6ac05362f8cb111f9

---

oh sorry :/ thanks for fixing this issue, @alexeykazakov !